crates/client/rpc/src/methods/get_block.rs

```diff
@@ -19,10 +17,10 @@ use starknet_core::types::{
 };
 
 use crate::utils::{
-    l1_gas_price, new_root, parent_hash, sequencer_address, starknet_version, status, timestamp, tx_conv,
-    tx_hash_compute, tx_hash_retrieve,
+    get_block_by_block_hash, l1_gas_price, new_root, parent_hash, sequencer_address, starknet_version, status,
+    timestamp, tx_conv, tx_hash_compute, tx_hash_retrieve,
 };
-use crate::Starknet;
+use crate::{Felt, Starknet};
 
 pub(crate) fn get_block_with_tx_hashes_finalized<A, B, BE, G, C, P, H>(
```

Comment:
Consider refactoring this into the newer `get_block_with_tx_hashes.rs` in `read/` to keep the code more organised.